.travis.yml

```diff
@@ -73,7 +73,9 @@ addons:
     packages:
     - g++-4.9
     - clang-3.9
-    update: true
+    # Updating the pkg repository causes Travis CI instances to fall flat on
```

Comment:
I would refrain from these comments. Just say what happened. Also, I dont see these failures, where do you see them:?

Reply:
Yeah, I was going to reword that to be a bit more positive..

I was seeing it in terms of failed CI runs on the other PRs. I'm going to try again with another branch as a control and see whether or not this is a "valid" solution.